File: StackSearch/StackSearch/Models/Items/Item.swift
//
//  Item.swift
//  StackSearch
//
//  Created by Saiful I. Wicaksana on 11/08/18.
//  Copyright © 2018 icaksama. All rights reserved.
//

import Foundation

internal struct Item: Codable {
    internal let tags: [String]
    internal let owner: Owner
    internal let is_answered: Bool
    internal let view_count: Int
    internal let answer_count: Int
    internal let score: Int
    internal let last_activity_date: Int
    internal let creation_date: Int
    internal let last_edit_date: Int?
    internal let question_id: Int
    internal let link: String
    internal let title: String
}

File: AStackSearch/app/src/main/java/com/icaksama/astacksearch/models/items/Item.java
package com.icaksama.astacksearch.models.items;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by icaksama on 13/08/18.
 */

public class Item {

    @SerializedName("tags")
    public List<String> tags;
    @SerializedName("owner")
    public Owner owner;
    @SerializedName("is_answered")
    public boolean is_answered;
    @SerializedName("view_count")
    public int view_count;
    @SerializedName("answer_count")
    public int answer_count;
    @SerializedName("score")
    public int score;
    @SerializedName("last_activity_date")
    public long last_activity_date;
    @SerializedName("creation_date")
    public long creation_date;
    @SerializedName("last_edit_date")
    public long last_edit_date;
    @SerializedName("question_id")
    public int question_id;
    @SerializedName("link")
    public String link;
    @SerializedName("title")
    public String title;
}